.github/workflows/lint-commits.yml

```diff
@@ -0,0 +1,11 @@
+name: Lint Commit Messages
+on: [pull_request]
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Do we really have to fetch all commits, tags and so on?

Reply:
The readme for the action I'm using specifies that this is needed so all commits on the PR are linted, not just the latest one. (https://github.com/wagoid/commitlint-github-action#usage)